src/shared/invite-response/invite-response.tsx

```diff
@@ -226,6 +244,285 @@ const InviteResponse: FC<InviteResponse> = ({
 								proposeNewTime={proposeNewTime}
 							/>
 					  )}
+
+				{invite[0]?.comp[0].loc && (
+					<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large' }}>
+						<Tooltip placement="left" label={t('tooltip.location', 'Location')}>
+							<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+								<Icon size="large" icon="PinOutline" />
+							</Row>
+						</Tooltip>
+
+						<Row takeAvailableSpace mainAlignment="flex-start">
+							<Tooltip placement="right" label={invite[0]?.comp[0]?.or.a}>
+								<Text size="medium" overflow="break-word" style={{ fontSize: '16px' }}>
+									{invite[0]?.comp[0].loc}
+								</Text>
+							</Tooltip>
+						</Row>
+					</Row>
+				)}
+
+				{chatLink && (
+					<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large' }}>
+						<Tooltip placement="left" label={t('tooltip.chat', 'Virtual Chat')}>
+							<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+								<Icon size="large" icon="VideoOutline" />
+							</Row>
+						</Tooltip>
+
+						<Row takeAvailableSpace mainAlignment="flex-start">
+							<Tooltip placement="right" label="link">
+								<LinkText
+									color="primary"
+									size="medium"
+									onClick={(): void => setMaxReqParticipantsToShow(requiredParticipants.length)}
+									overflow="break-word"
+								>
+									Chat&#39;s room
+								</LinkText>
+							</Tooltip>
+						</Row>
+					</Row>
+				)}
+
+				{haveEquipment && (
+					<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large', bottom: 'small' }}>
+						<Tooltip placement="left" label={t('tooltip.equipment', 'Equipment')}>
+							<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+								<Icon size="large" icon="BriefcaseOutline" />
+							</Row>
+						</Tooltip>
+						<Row takeAvailableSpace mainAlignment="flex-start">
+							<Tooltip placement="right" label="Equipment name">
+								<Text size="medium" overflow="break-word" style={{ fontSize: '16px' }}>
+									Equipment name
+								</Text>
+							</Tooltip>
+						</Row>
+					</Row>
+				)}
+
+				<Row
+					width="100%"
+					mainAlignment="flex-start"
+					crossAlignment="flex-start"
+					padding={{ vertical: 'medium' }}
+				>
+					<Row width="50%" mainAlignment="flex-start" crossAlignment="flex-start">
+						<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+							<Icon size="large" icon="PeopleOutline" />
+						</Row>
+						<Row takeAvailableSpace mainAlignment="flex-start" crossAlignment="flex-start">
+							<Row mainAlignment="flex-start" width="100%" padding={{ bottom: 'extrasmall' }}>
+								<Text overflow="break-word">
```

Comment:
you forgot to pass the default value here ,an example can be
t('reminder.hour_before', {
			count: 5,
			defaultValue: '{{count}} hour before',
			defaultValue_plural: '{{count}} hours before'
		}),